updater/lib/dependabot/setup.rb

```diff
@@ -37,7 +37,31 @@
     pub
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = lambda do |event, hint|
+    debugger
```

Comment:
TODO: need to get this to fire so I can debug this section of code, and see what the payloads look like...